### PR TITLE
update attr version to 2.4.48

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/attr/package.py [hunk 1]
```diff
@@ -23,4 +31,7 @@ def configure_args(self):
     # Ref. http://www.linuxfromscratch.org/blfs/view/7.5/postlfs/attr.html
     def install(self, spec, prefix):
         make()
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/attr/package.py]: Any idea why `make()` is present in the `install` phase? We already run it during the `build` phase.

Comment (darmac) [reply var/spack/repos/builtin/packages/attr/package.py]: I've no idea about that, I think maybe it's a history issue :(

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/attr/package.py]: If you remove it does it still install correctly?

Comment (darmac) [reply var/spack/repos/builtin/packages/attr/package.py]: I tried remove it, and attr@2.4.47 and attr@2.4.48 can both build well.
And I uploaded this patch: https://github.com/spack/spack/pull/15535/commits/ad5d82a0cb72d25dc86bd53d4b3efee8186be3dd

**Pull request merged**